aws_asdi_pipelines/pipelines/amazonia_1/app.py

```diff
@@ -0,0 +1,81 @@
+"""Amazonia1 asdi handler."""
+
+import json
+import os
+import re
+
+import requests
+from aws_lambda_powertools.utilities.data_classes import SQSEvent, event_source
+from stactools.amazonia_1 import create_item
+from stactools.core import use_fsspec
+
+from aws_asdi_pipelines.cognito.utils import get_token
+
+
+def xml_key_from_quicklook_key(key: str) -> str:
```

Comment:
@fredliporace It looks like this function might need to be extended for some different cases of inferring the xml key.  As an example looking at `s3://amazonia-pds/AMAZONIA1/WFI/033/018/AMAZONIA_1_WFI_20230621_033_018_L4/`. 

The xml key is actually `AMAZONIA_1_WFI_20230621_033_018_L4_LEFT_BAND2.xml` rather than the key `AMAZONIA_1_WFI_20230621_033_018_L4_BAND2.xml` which the function produces.

Reply:
@sharkinsspatial see #34, working on that.

Reply:
@sharkinsspatial #34 is fixed by bdfe86c but my testing in actual aws environment is limited, let me know if any additional work is required.